storage/src/vespa/storage/storageserver/rpc/message_codec_provider.h

```diff
@@ -34,14 +31,12 @@ class MessageCodecProvider {
     mutable std::shared_mutex _rw_mutex;
     std::shared_ptr<WrappedCodec> _active_codec;
 public:
-    MessageCodecProvider(std::shared_ptr<const document::DocumentTypeRepo> doc_type_repo,
-                         std::shared_ptr<const documentapi::LoadTypeSet> load_type_set);
+    MessageCodecProvider(std::shared_ptr<const document::DocumentTypeRepo> doc_type_repo);
```

Comment:
Consider adding `explicit` to `WrappedCodec` and `MessageCodecProvider` ctors

Reply:
Fixed